projects/kasu/index.js

```diff
@@ -0,0 +1,69 @@
+const { cachedGraphQuery } = require('../helper/cache')
+const ADDRESSES = require('../helper/coreAssets.json')
+
+const GRAPH_URLS = {
+  base: {
+    uri:  'https://subgraph.satsuma-prod.com/3ed46ea711d3/kasu-finance--314476/kasu-base/api',
+    query: `{
+              lendingPools (where:{isStopped: false}){
+                id
+                balance
+              }
+            }`
+  },
+  plume: {
+    uri: 'https://api.goldsky.com/api/public/project_cm9t3064xeuyn01tgctdo3c17/subgraphs/kasu-plume/prod/gn',
+    query: `{
+              lendingPools {
+                id
+                balance
+              }
+            }`
+  }
+};
+
+const CHAIN_ASSET = {
+  base: {
+    asset: ADDRESSES.base.USDC,
+    decimals: 6
+  },
+  plume: {
+    asset: ADDRESSES.plume_mainnet.pUSD,
+    decimals: 6
+  },
+}
+
+function offChainPoolTVL(api, poolAddress) {
+  return api.call({
```

Comment:
can we use api.multiCall() here instead of Promise.all(call()) please

Reply:
done